source/opt/local_access_chain_convert_pass.cpp

```diff
@@ -236,6 +241,16 @@ void LocalAccessChainConvertPass::FindTargetVars(Function* func) {
   }
 }
 
+bool LocalAccessChainConvertPass::HasNoOpenCL100DebugRef(
+    Instruction* inst) const {
+  return !get_def_use_mgr()->WhileEachUser(inst, [](Instruction* user) {
```

Comment:
Is this returning what I expect?  You return true if it has a debug reference, but I expect it to return false because the name is "Has no opencl 100 debug ref."

Reply:
Likewise.